yt_idefix/data_structures.py

```diff
@@ -759,3 +788,109 @@ def _is_valid(cls, filename: str, *args, **kwargs) -> bool:
 
     def _get_log_file(self) -> str:
         return os.path.join(self.directory, "vtk.out")
+
+
+class PlutoXdmfDataset(StaticPlutoDataset):
+    _dataset_type = "pluto-xdmf"
+    _index_class = PlutoXdmfHierarchy
+    _field_info_class = PlutoXdmfFields
+
+    def __init__(
+        self,
+        filename,
+        *,
+        eos_params: dict[str, float] | None = None,
+        **kwargs,
+    ):
+        super().__init__(filename, **kwargs)
```

Comment:
Yes sure. That I understand. What I'm suggesting is we can retain this basic `__init__()` structure here so that we can add any new feature for later and the path size will be smaller. If you don't want this then I can remove this redundant part as well.

Reply:
In general I don't want to add code unless it's immediately useful. 
Here's it's actually worse than useless because it introduces an unused keyword argument, so I'd like this removed for now.